model-providers/bedrock/deployment/src/test/java/io/quarkiverse/langchain4j/bedrock/deployment/BedrockAntrophicStreamingChatModelTest.java

```diff
@@ -1,80 +0,0 @@
-package io.quarkiverse.langchain4j.bedrock.deployment;
-
-import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
-import static com.github.tomakehurst.wiremock.client.WireMock.anyUrl;
-import static com.github.tomakehurst.wiremock.client.WireMock.post;
-import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
-import static io.quarkiverse.langchain4j.bedrock.deployment.BedrockStreamHelper.createCompletion;
-import static io.quarkiverse.langchain4j.bedrock.deployment.BedrockStreamHelper.decode;
-import static org.assertj.core.api.Assertions.assertThat;
-
-import java.util.List;
-
-import jakarta.inject.Inject;
-
-import org.jboss.shrinkwrap.api.ShrinkWrap;
-import org.jboss.shrinkwrap.api.spec.JavaArchive;
-import org.junit.jupiter.api.DisplayNameGeneration;
-import org.junit.jupiter.api.DisplayNameGenerator;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.RegisterExtension;
-
-import dev.langchain4j.model.bedrock.BedrockAnthropicStreamingChatModel;
-import dev.langchain4j.model.chat.StreamingChatModel;
-import io.quarkus.arc.ClientProxy;
-import io.quarkus.test.QuarkusUnitTest;
-
-@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
-class BedrockAntrophicStreamingChatModelTest extends BedrockTestBase {
```

Comment:
Why was thid deleted?

Reply:
Because the `BedrockAntrophicStreamingChatModel` class doesn't exist anymore.

Reply:
Got it, thanks